anchor/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "anchor"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anchor-lang = { version = "0.29.0", optional = true}
+borsh = "0.10"
```

Comment:
is this needed? It is recursive because `anchor-lang` already depends on `borsh`

Reply:
not sure...it was there in the old `cargo.toml`. Maybe @caballa knows?

Reply:
try removing it :)